app/start.py

```diff
@@ -13,15 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import urlparse
 import const
-from utils import BaseHandler
+import utils
 from model import Counter
 
 
-class Handler(BaseHandler):
+class Handler(utils.BaseHandler):
     repo_required = False
+
     def get(self):
         self.env.robots_ok = True
+        self.env.amp_url = self.set_amp_url_param(self.request.url, 'lang', self.env.lang)
```

Comment:
You can probably simply call self.get_url('/amp_start') instead of self.set_amp_url_param(...)?